sample-queries/sample-queries.json

```diff
@@ -1981,6 +1981,76 @@
         "postBody": "{\r\n  \"applicationName\": \"LinkedResource created from Microsoft Graph Explorer\"\r\n}",
         "tip": "This query requires the Tasks.ReadWrite permission. To find a value for the taskListId parameter, you can run: GET https://graph.microsoft.com/beta/me/todo/lists. to find a value for the taskId, you can run: GET https://graph.microsoft.com/beta/me/todo/lists/{taskListId}",
         "skipTest": false
+    },
+    {
+        "id": "fd58d371-315e-49d8-aa74-58fc4ad1337c",
+        "category": "Identity and Access",
+        "method": "GET",
+        "humanName": "list Azure ad gallery app",
+        "requestUrl": "/beta/applicationTemplates",
+        "docLink": "https://docs.microsoft.com/graph/api/applicationtemplate-list?view=graph-rest-beta",
+        "tip": "We’d like to hear from you. Please leave your feedback on this API here: https://aka.ms/appTemplateAPISurvey",
```

Comment:
@bettirosengugi - thoughts on adding the `feedback` message on the `tip` property?

Reply:
This is fine.